docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -118,9 +118,20 @@ Here, we use the `mysql` and `pymysql` dialects to set up an SSL connection to a
    ```toml
    sources.sql_database.credentials="mssql://user:pw@my_host/my_database?trusted_connection=yes"
```

Comment:
this looks wrong. user and password should not be specified. also the dialect is missing from scheme.
https://stackoverflow.com/questions/24085352/how-do-i-connect-to-sql-server-via-sqlalchemy-using-windows-authentication
```
from sqlalchemy import create_engine

server = <your_server_name>

database = <your_database_name>

engine = create_engine('mssql+pyodbc://' + server + '/' + database + '?trusted_connection=yes&driver=ODBC+Driver+17+for+SQL+Server')
```
```